robota_core/repository.py

```diff
@@ -147,6 +173,9 @@ def _diff_from_github(self, diff_info: github.File.File):
             self.new_file = False
         self.diff = diff_info.patch
 
+        if diff_url is not None:
+            # no way to link to the exact file in github UI
```

Comment:
The hashes are SHA256 of the file names. I remember there being _some_ edge case where the anchors did not work, however, and I had to drop the entire thing. I'll try to figure out what the issue was, probably something to do with very large diffs.

Reply:
Oh great - so we can get the hashes.  That's super useful to know, thanks.

If you can manage to get back to those edge cases, that would be good.

Reply:
Does clicking on the `pyproject.toml` link direct you to the file? When I click it, it directs me to the commits tab rather than the file. Same with other files I've tried. The two commit links work fine.